Source/FakeItEasy/Core/BootstrapperLocator.cs

```diff
@@ -25,7 +25,9 @@ public static IBootstrapper FindBootstrapper()
             var bootstrapperInterface = typeof(IBootstrapper);
 
             var appDomainAssemblies = AppDomain.CurrentDomain.GetAssemblies();
-            var appDomainAssembliesReferencingFakeItEasy = appDomainAssemblies.Where(assembly => assembly.ReferencesFakeItEasy());
+            var appDomainAssembliesReferencingFakeItEasy = appDomainAssemblies
+                .Where(assembly => !assembly.IsDynamic)

```

Comment:
no Assembly.IsDynamic in .NET 3.5.
Check out https://github.com/FakeItEasy/FakeItEasy/blob/bd57a05c0de5ff37609230fd1b8bc4ec45bc9d4d/Source/FakeItEasy/Core/TypeCatalogue.cs#L64